src/charm.py

```diff
@@ -89,6 +89,11 @@ def _on_install(self, _) -> None:
             self._set_status(Status.SNAP_NOT_INSTALLED)
             return
 
+        # TODO: run on install
```

Comment:
**question**: does it need to run post upgrade as well?

Reply:
Turns out doesn't need to run on install. This part is now moved and will no-op if dirs are already formatted